portfolio/src/main/webapp/comments.css

```
@@ -89,3 +89,12 @@ small {
 #auth-button.logout:hover {
   background-color: rgb(146, 7, 0);
 }
+
+.user-email {
+  font-weight: bold;
+  color: rgb(0, 130, 0)
```

```suggestion
  color: rgb(0, 130, 0);
  font-weight: bold;
```
nit: alphabetize declarations, also missing semicolon
https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order